src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+abstract class AbstractFigure implements Drawable, Area {
```

Comment:
It's clear from class definition that AbstractFigure is abstract class
```suggestion
abstract class Figure implements Drawable, Area {
```